docker-compose.yml.tmpl

```diff
@@ -128,16 +111,19 @@ services:
 
   anthias-nginx:
     image: screenly/anthias-nginx:${DOCKER_TAG}-${DEVICE_TYPE}
-    build:
-      context: .
-      dockerfile: docker/Dockerfile.nginx
     ports:
       - 80:80
     environment:
       - HOME=/data
     depends_on:
       - anthias-server
       - anthias-websocket
+    networks:
+      default:
+      anthias-extra-network-1:
+        aliases:
+          - 'anthias-nginx'
+          - 'srly-ose-nginx'
```

Comment:
Maybe add a comment to remove the changes once a new version of the WebView has been released.